app/assets/javascripts/discourse/components/add-category-tag-classes.js.es6

```diff
@@ -13,20 +13,22 @@ export default Ember.Component.extend({
       return;
     }
     const slug = this.get("category.fullSlug");
+    const tags = this.get("tags");
+
     this._removeClass();
-    if (slug) {
-      $("body").addClass(`category-${slug}`);
-    }
+
+    if (slug) $("body").addClass(`category-${slug}`);
```

Comment:
I think it would be much faster to only do one call to `addClass`. Something like

```javascript
let classes = [];
if (slug) classes.push(`category-${slug}`);
if (tags) tags.forEach(t => classes.push(`tag-${t}`));
if (classes.length > 0) $("body").addClass(classes.join(" "));
```